ui/src/app/pages/access/access-renewal-page.tsx

```diff
@@ -116,8 +173,17 @@ export const AccessRenewalPage = fp.flow(
 
   return <FadeBox style={{margin: '1rem auto 0', color: colors.primary}}>
     <div style={{display: 'grid', gridTemplateColumns: '1.5rem 1fr', alignItems: 'center', columnGap: '.675rem'}}>
-      <BackArrow style={{height: '1.5rem', width: '1.5rem'}}/>
-      <div style={styles.h1}>Yearly Researcher Workbench access renewal</div>
+
+      {maybeDaysRemaining(profile) < 0
+        ? <React.Fragment>
+            <ExclamationTriangle style={{height: '1.5rem', width: '1.5rem'}}/>
+            <div style={styles.h1}>Access to the Researcher Workbench revoked.</div>
```

Comment:
Maybe "has been revoked" ?  Or did we decide to soften this wording a bit recently?

Reply:
This will be softened - any tweaks I will make in one of the follow up PR's